lib/App/DuckPAN/Web.pm

```diff
@@ -539,6 +541,19 @@ sub request {
 	return $response;
 }
 
+
+sub run_on_ready_script {
+	my ($str) = @_;
+
+	# any unescaped script tags at this stage will XSS the page - so strip them
+	$str =~ s|</?script>||gi;
```

Comment:
Content inside should be fine, we really just need to remove any `</script>` tags so that the `<script>` tag the JSON is inside of doesn't get closed prematurely.